src/main/webapp/app/views/inputtype/input-strict_contact.html

```diff
@@ -0,0 +1,32 @@
+<div class="btn-group btn-group-justified submission-input-select" uib-dropdown keyboard-nav role="group">
+  <button type="button" class="btn dropdown-toggle dropdown-toggle-button" uib-dropdown-toggle>
+    <span class="caret"></span>
+  </button>
+  <button ng-class="{selected: fieldValue.value}" type="button" class="btn btn-default dropdown-text" disabled>
+    <span class="text-left dropdown-text-label">{{fieldValue.value || 'Choose ' + profile.gloss}}</span>
+  </button>
+  <ul class="dropdown-menu" uib-dropdown-menu role="menu">
+    <li class="select-option"
+      role="menuitem"
+      ng-click="fieldValue.value = word.name; fieldProfileForm.$dirty = true; saveWithCV(fieldValue, word);" 
```

Comment:
This might be better served with the ```field.value = ...``` inside of a single function.
(and add the appropriate tests.)

Reply:
These templates are used in the field profile directive and there isn't an affordance for passing a function for this. I followed the pastern seen in the standard input contact.